### DISCUSSION TITLE
Some suggestions for the Calendars plug-in

### DISCUSSION THREAD

**Discussion by kajaji** [Stellarium/stellarium#1566] [Translations]
These are some suggestions to improve the readability of the Calendars plug-in:

- The link provided here is:
 https://github.com/Stellarium/stellarium/blob/0f12e9b2a8ec89543be803c0ff1676ef3546d133/plugins/Calendars/src/gui/CalendarsDialog.cpp#L206
is not working. I found the correct link as:
https://doi.org/10.1017/9781107415058

- I suggest for this line:
return QString("%1, %2 - %3 (%4) - %5 A.H.") 
here: https://github.com/Stellarium/stellarium/blob/0f12e9b2a8ec89543be803c0ff1676ef3546d133/plugins/Calendars/src/IslamicCalendar.cpp#L93
to add "A.H." to the translation strings with guidance for translators that A.H. stands for "after hegira" meaning the year prophet Mohammad (Peace be upon him and all prophets) migrated from Mecca to Madinah.

- Similarly for:
return QString("%1, %2 - %3 (%4) - %5 A.P.")
here:
https://github.com/Stellarium/stellarium/blob/0f12e9b2a8ec89543be803c0ff1676ef3546d133/plugins/Calendars/src/PersianArithmeticCalendar.cpp#L95
to add "A.P." to the translation strings with guidance for translators that A.P. means Anno Persico/Anno Persarum = Persian year

- Similarly for the Hindu Calendars 
return QString("%1, %2 - %3 (%4) %5- %6 K.Y.")
here:
https://github.com/Stellarium/stellarium/blob/0f12e9b2a8ec89543be803c0ff1676ef3546d133/plugins/Calendars/src/OldHinduLuniSolarCalendar.cpp#L86
to add "K.Y." to the translation strings with guidance for translators that K.Y. stands for Kali Yuga.

- Calendars display in right-to-left languages:
The calendar columns are displayed left-to right. If the calendar name column can be displayed first (to the right), it will look better and more logical for right-to-left language users.

![image](https://user-images.githubusercontent.com/34771982/112333119-5f4c0780-8ccb-11eb-8eb1-e0313303b2d2.png)

Thanks

**Comment by alex-w** [comment]
Please check the fresh version (development snapshot) of Stellarium:
https://github.com/Stellarium/stellarium-data/releases/tag/weekly-snapshot
